NuSpec/Build/RedGate.AppHost.targets

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup Condition="$(RedGateAppHostOutputPath) == ''">
+        <RedGateAppHostOutputPath>$(TargetDir)</RedGateAppHostOutputPath>
+    </PropertyGroup>
+    <Target Name="CopyRedGateAppHostFiles" AfterTargets="Build">
+        <ItemGroup>
+            <RedGateAppHostFiles Include="$(MSBuildThisFileDirectory)\..\tools\**\*.*" />
+        </ItemGroup>
+        <Copy SourceFiles="@(RedGateAppHostFiles)" DestinationFiles="@(RedGateAppHostFiles->'$(RedGateAppHostOutputPath)\%(RecursiveDir)%(Filename)%(Extension)')" SkipUnchangedFiles="true" />
```

Comment:
Looks like tools might be the wrong folder at the very least. Its meant for tools accessible from the package manager console:
http://docs.nuget.org/ndocs/create-packages/creating-a-package#from-a-convention-based-working-directory